src/local/butler/reproduce.py

```diff
@@ -100,11 +103,10 @@ def _download_testcase(testcase_id, testcase, configuration):
     raise errors.ReproduceToolUnrecoverableError(
         'Unable to download test case.')
 
-  # Create a temporary directory where we can store the test case.
   bot_absolute_filename = response['x-goog-meta-filename']
-  testcase_directory = os.path.join(
-      environment.get_value('ROOT_DIR'), 'current-testcase')
-  shell.create_directory(testcase_directory)
+  # Store the test case in the config directory for debuggability.
+  testcase_directory = os.path.join(CONFIG_DIRECTORY, 'current-testcase')
+  shell.remove_directory(testcase_directory, recreate=True)
```

Comment:
shell.remove_directory just does os.mkdir, probably need to change to os.makedirs so that intermediates are created.

Reply:
Good catch. Changed in shell.remove_directory.